Solves #298

pdias opened the pull request:
Solves:

User Deprecated: The "doctrine.orm.default_entity_manager" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public, or stop using the container directly and use dependency injection instead.

This warning is caused by the JMSDIExtraBundle proxy replacing the Doctrine Service.
JMSDIExtraBundle, does not properly preserve the service visibility when overwriting the service.

pdias committed:
Solves #298

Solves:

User Deprecated: The "doctrine.orm.default_entity_manager" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public, or stop using the container directly and use dependency injection instead.

This warning is caused by the JMSDIExtraBundle proxy replacing the Doctrine Service.
JMSDIExtraBundle, does not properly preserve the service visibility when overwriting the service.

pdias committed:
Compatibility with older SF versions

Method isPrivate don't exist in Symfony\Component\DependencyInjection\DefinitionDecorator in versions prior to 2.3

greg0ire reviewed (commented):
I don't get it... Being private is just the opposite of being public, right? Why are there 2 methods?

pdias committed:
Compatibility with older SF versions 

Method isPrivate don't exist in Symfony\Component\DependencyInjection\DefinitionDecorator in versions prior to 2.3

pdias commented:
@greg0ire You're right ... I checked ... and:

```
public function setPublic($boolean)
    {
        $this->changes['public'] = true;

        $this->public = (bool) $boolean;
        $this->private = false;

        return $this;
    }
```

I'm going to change.

pdias committed:
Compatibility with older SF versions

Method isPrivate don't exist in Symfony\Component\DependencyInjection\DefinitionDecorator in versions prior to 2.3

GuilhemN commented:
Thank you @pdias.

ZORRO97 commented:
I updated my project with 1.9.1 version and the deprecation for doctrine.orm.default_entity_manager still appears.
Any ideas for solve this problem ?

ZORRO97 commented:
It's solved when I require dev-master version. I'm waiting for a stable version. Will this arrive soon ?
